src/pages/Details.vue

```diff
@@ -313,6 +315,7 @@ export default {
         return {
             page: 1,
             perPage: 25,
+            timeRemaining: 0,
```

Comment:
hey so if i just make a new component and pass in the time remaining would that suffice

Reply:
I think passing when the countdown should end (daatetime) would be a nicer api

Reply:
Hey this would be difficult as there are many cases (then where we would have to update the time instead of interval by converting) which we would have to handle all that converstion from seconds to date and then from date to seconds.... isnt it a lil too uncessary?